Tests/Tests.csproj

```diff
@@ -69,10 +72,15 @@
     </Compile>
   </ItemGroup>
   <ItemGroup>
+    <ProjectReference Include="..\AutoUpdate\CKAN-autoupdate.csproj" />
+    <ProjectReference Include="..\Cmdline\CKAN-cmdline.csproj" />
+    <ProjectReference Include="..\ConsoleUI\CKAN-ConsoleUI.csproj" />
     <ProjectReference Include="..\Core\CKAN-core.csproj" />
+    <ProjectReference Include="..\GUI\CKAN-GUI.csproj" />
+    <ProjectReference Include="..\Netkan\CKAN-netkan.csproj" />
   </ItemGroup>
   <Choose>
-    <When Condition=" '$(TargetFramework)' == 'net45' ">
+    <When Condition=" '$(TargetFramework)' == 'net462' ">
```

Comment:
Same as above

Reply:
Fixed: typo from original attempt at 462, now 472. Tests still at 852 of 860 & builds. Due apologies!